templates/zerver/app/navbar_alerts.html

```diff
@@ -50,14 +50,27 @@
     </div>
     <div data-process="bankruptcy" class="alert alert-info brankruptcy">
         <div data-step="1">
-            {% trans count=page_params.unread_msgs.count %}
-            Welcome back!  You have <span class="bankruptcy_unread_count">{{ count }}</span> unread messages.  Do you want to mark them all as read?
-            <span class="buttons">
-                <a class="alert-link accept-bankruptcy" role="button" tabindex=0>Yes, please!</a>
-                &bull;
-                <a class="alert-link exit" role="button" tabindex=0>No, I'll catch up.</a>
-            </span>
-            {% endtrans %}
+            {% with count=page_params.unread_msgs.count %}
+                {% if count > 50000 %}
+                    {% trans %}
+                    Welcome back!  You have more than <span class="bankruptcy_unread_count">{{ count }}</span> unread messages. Zulip only shows first 50000 unread messages. Do you want to mark them all as read?
```

Comment:
Let's just fork the one line, and just have it say "You have ..." or "You have at least ..." as the two options.  

Also, the structure of this translateable string is super weird -- we should have smaller translation tag area that minimizes the amount of HTML squashed between unrelated sentence that we're making translators deal with.

@hackerkid want to help and/or do a quick prep PR to clean up the way translation tags are done here?

Reply:
(I suppose we would make "more than 50000" a link, to a /help/ page that I can write explaining what happens when you get there)

Reply:
Created #17641